qa/python_models/string/model.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import numpy as np
+import sys
+import json
+
+sys.path.append('../../')
+import triton_python_backend_utils as pb_utils
+
+
+class TritonPythonModel:
+
+    def initialize(self, args):
+        self.model_config = json.loads(args['model_config'])
+
+    def execute(self, requests):
+        """ This function is called on inference request.
+        """
+
+        responses = []
+        for request in requests:
+            input_tensors = request.inputs()
+            in_0 = pb_utils.get_input_tensor_by_name(request, "INPUT0")
+            unicode_character = '😀'
+            if str(in_0.as_numpy().tobytes(), encoding='utf-8') == unicode_character:
+                out_tensor_0 = pb_utils.Tensor("OUTPUT0", np.array(['success'], dtype=np.bytes_))
+            else:
+                out_tensor_0 = pb_utils.Tensor("OUTPUT0", np.array(['failure'], dtype=np.bytes_))
+            responses.append(
+                pb_utils.InferenceResponse([out_tensor_0]))
+        return responses
```

Comment:
Add new line after

Reply:
It does have a new line. Whenever it doesn't, it shows a little sign at the end of the line.